source/server/listener_impl.cc

```diff
@@ -32,6 +32,10 @@
 
 #ifdef ENVOY_ENABLE_QUIC
 #include "source/common/quic/active_quic_listener.h"
+#include "source/common/quic/gso_defines.h"
+#endif // ENVOY_ENABLE_QUIC
+
+#ifdef UDP_GSO_BATCH_WRITER_COMPILETIME_SUPPORT
```

Comment:
If you #ifdef UDP_GSO_BATCH_WRITER_COMPILETIME_SUPPORT in udp_gso_batch_writer.h, this ifdef is not needed.

Reply:
well right now the file is compile-selected out in the build.
we could leave it in the build and #ifdef out the whole contents, but this seems cleaner to me?

Reply:
also I'm _super_ baffled by windows breakage

source/server/listener_impl.cc(39): fatal error C1083: Cannot open include file: 'source/common/quic/udp_gso_batch_writer.h': No such file or directory


but it should only see that if UDP_GSO_BATCH_WRITER_COMPILETIME_SUPPORT is 1 which should only happen on linux? What am missing? :-/

Reply:
oops nm, PEBKAC, fixed.

Reply:
Agreed this is cleaner. I'm fine either way.
BTW, udp_gso_batch_writer.h is already ifdef'ing out the whole contents, isn't it?

Reply:
uggggh, you're totally right, which means I've been attacking the wrong problem from the start.
Thanks for saving me the commit + mobile update + realizing this didn't solve my problem 
trying a new workaround per your offline suggestions!